packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -5,11 +5,133 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import '../../globals/imports';
-
 @mixin leadspace {
   .#{$prefix}--leadspace {
-    background-color: transparent;
+    position: relative;
+    padding-top: carbon--rem(370px) / carbon--mini-units(40) * 100%;
+    height: 0;
+    background: $ui-background;
+
+    &__image {
+      object-fit: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    &__container {
+      position: absolute;
+      top: 0;
+      height: 100%;
+      width: 100%;
+    }
+
+    &__overlay {
+      @include carbon--make-container;
+
+      padding-top: $layout-03;
+      padding-bottom: $layout-03;
+      position: absolute;
```

Comment:
The biggest problem I'm seeing with the grid right now is it's left aligned because of this `position: absolute` and a couple of the properties below it. You might `position: absolute` on a parent, or have the grid container start as a child of this block.


*It should look like this...*
![image](https://user-images.githubusercontent.com/3793636/65060728-0b29cf80-d93e-11e9-92d8-903755531e7d.png)

*This is what I'm seeing...*
![image](https://user-images.githubusercontent.com/3793636/65060783-2399ea00-d93e-11e9-9556-90d2046edb05.png)

Reply:
@photodow I think we are good to go now with the package updates in